queue_services/entity-emailer/tests/unit/email_processors/test_filing_notification.py

```diff
@@ -67,14 +85,14 @@ def test_correction_incorporation_notification(app, session, status, has_name_ch
         email = filing_notification.process(
             {'filingId': filing.id, 'type': 'correction', 'option': status}, token)
         if status == 'PAID':
-            assert 'comp_party@email.com' in email['recipients']
+            assert testing_email in email['recipients']
             assert email['content']['subject'] == 'Confirmation of Correction of Incorporation Application'
             assert 'Incorporation Application (Corrected)' in email['content']['body']
```

Comment:
I only changed it here because before I changed anything the test was failing. I checked the conditions and it seemed to be a wrong asserting for 'comp_party@email.com'.

Reply:
Hey Andre, can you double check with the BAs whether or not the completing party is supposed to get emailed for a correction to the incorporation filing? If so it needs to get added in the code (same as for a normal incorporation filing). Otherwise can you change it to be ```comp_party@email.com not in in email['recipients']```

Reply:
this assert is just for testing the completing party notification -- the testing_email is asserted lower down already

Reply:
Ok. Thanks! I'm waiting for BA's asnwer.